editor/test/integration/particlefx_test.clj

```diff
@@ -59,9 +53,9 @@
           fetch-anim-fn (fn [index] (get emitter-sim-data index))
           transforms [(doto (Matrix4d.) (.setIdentity))]
           sim (plib/make-sim 16 256 prototype-msg transforms)
-          attribute-infos (into []
-                                (map graphics/attribute-key->default-attribute-info)
-                                [:position])
+          attribute-infos [(-> :position
+                               (graphics/attribute-key->default-attribute-info)
+                               (assoc :coordinate-space :coordinate-space-world))]
```

Comment:
The `:coordinate-space` of the `:position` attribute is `:default` in `attribute-key->default-attribute-info`. This is replaced by either `:coordinate-space-local` or `:coordinate-space-world` in `graphics/shader-bound-attributes` dependent on the `:vertex-space` of the used material. Here, we just make it up.